doc/Development_Documentation/03_Documents/01_Editables/02_Areablock/README.md

```diff
@@ -202,6 +204,28 @@ The manual mode offers you the possibility to use areablocks with custom HTML, t
 <?php $areaBlock->end(); ?>
 ```
 
+```twig
+{% set areaBlock = pimcore_areablock("myArea", {"manual":"true"}) %}
+
+{% do areaBlock.start %}
```

Comment:
Should we add parenthesis to these calls, as they're actual function calls?

```suggestion
{% do areaBlock.start() %}
```

Reply:
Done :)